src/components/molecules/EntityGrid/EntityGrid.styled.tsx

```diff
@@ -16,7 +16,7 @@ export const StyledEntityGrid = styled.div<{$columns?: number; $itemWidth: numbe
 
 export const ItemWrapper = styled.div`
   display: flex;
-  justify-content: space-between;
+  align-items: space-between;
```

Comment:
There is no such thing as `align-items: space-between` - we may delete it then :)